functions/nodejs-apps.bash

```diff
@@ -54,8 +54,9 @@ frontail_setup() {
 nodered_setup() {
   echo -n "$(timestamp) [openHABian] Installing Node-RED... "
   FAILED=0
-  cond_redirect wget -O /tmp/update-nodejs-and-nodered.sh https://raw.githubusercontent.com/node-red/raspbian-deb-package/master/resources/update-nodejs-and-nodered || FAILED=1
+  cond_redirect wget -O /tmp/update-nodejs-and-nodered.sh https://raw.githubusercontent.com/node-red/linux-installers/master/deb/update-nodejs-and-nodered || FAILED=1
   cond_redirect chmod +x /tmp/update-nodejs-and-nodered.sh || FAILED=1
+  cond_redirect ln -sf /tmp/update-nodejs-and-nodered.sh /usr/bin/update-nodejs-and-nodered || FAILED=1
```

Comment:
What is the purpose of this? You link a file in tmp to /usr/bin which will probably be gone after reboot (tmp dir purged).
If you want to keep the script, it should be put moved to /usr/local/sbin.
Install works fine without this command as script from new repo does not issue the rm command in the last line.
Just did the install on my pi.

Reply:
Several people to complain about nodered not installing. 

> What is the purpose of this?
See bottom of #738,

Reply:
Sorry, I do not get it.
The script from the old repo had a call to rm in the last line, which did obviously break the install since the file was not there.
Switching to the new repo this is no longer part of the script.
Or did I overlook something?

Reply:
Ok I didn't compare the nodered install scripts from those repos hence I thought the rm line is still in. You are correct it isn't. Will only merge the repo change.